Sources/swift-type-adoption-reporter/main.swift

```diff
@@ -65,15 +65,15 @@ do {
 
     guard let types = parsedArguments.get(typesArgument) else {
         fputs("Missing types to report on.\n", stderr)
-        exit(1)
+        exit(EXIT_FAILURE)
```

Comment:
Just changing the hardcoded 0/1 exit codes to these standard constants I discovered, since I figure that's the preferred best practice